src/Agents/Adapters/Anthropic.php

```
@@ -457,4 +457,19 @@ protected function formatErrorMessage($json): string
 
         return '('.$errorType.') '.$errorMessage;
     }
+
+    public function getSupportForEmbeddings(): bool
+    {
+        return false;
+    }
+
+    public function embed(string $text): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$text'. (undefined)

(UnusedFormalParameter)